Dockerfile

```diff
@@ -29,7 +29,7 @@
 # 3. Github Actions have shared networking configured, so we need to set a random
 # SCCACHE_SERVER_PORT port to avoid conflicts in case of parallel compilation
 
-ARG ALPINE_VERSION=3.16
+ARG ALPINE_VERSION=3.18
```

Comment:
Why did you change the version of alpine?

Reply:
It was changed in the buildbase image

Reply:
This change is necessary because Alpine 3.16 only packages Node 16.

Reply:
According https://www.alpinelinux.org/posts/Alpine-3.18.0-released.html, it should ship node.js 20 (when installed with apk i believe), but in fact its node v18, here is output from dapi container:


```
/platform $ node -v
v18.17.1
/platform $ ```
> NAME="Alpine Linux"
> ID=alpine
> VERSION_ID=3.18.3
> PRETTY_NAME="Alpine Linux v3.18"
> HOME_URL="https://alpinelinux.org/"
> BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
> /platform $ node -v
> v18.17.1
```

Reply:
We also changed it here in the buildbase https://github.com/strophy/docker-buildbase/blob/master/Dockerfile (in the image 0.0.9)

Reply:
oh ok, i just wondering if it works properly.